HW0/P6/P6.txt

```diff
@@ -1 +1,86 @@
-# Your answers here
+Problem 6.
```

Comment:
@zezhouliu is there something in my code that could be causing the noise (I was using VirtualBox)? Given the fluctuations, I could have easily data snooped and chosen a graph that I would expect would be most representative of the trend (indeed, I had several graphs that look similar to those of my peers), but that would be unscientific and misleading. This is why I chose to implement error bars and repetitions here. 

The distribution of standard errors itself is very indicative of a trend in performance. Narrow error bars at 1e-6 s and above 1e-2.5 s wait times are indicative of clear performance differences between the parallel and serial methods, and there is an obvious transition zone between these two values. The wide error bars in between these two wait times may suggest that the speedup values at these wait times are sensitive to fluctuations in bandwidth or other components inherent to my computer or other concurrent processes (if not my code, possibly due to VirtualBox, my computer, etc., although I made it a point to kill as many irrelevant running tasks on my computer prior to running this). The best way to better determine the exact interval in which the transition occurs would have been to increase the number of reps of my run.